lock-api-objects/src/main/java/com/palantir/lock/watch/CommitUpdate.java

```diff
@@ -39,6 +40,7 @@ static CommitUpdate invalidateAll() {
         return ImmutableInvalidateAll.builder().build();
     }
 
+    @Unsafe
```

Comment:
Why do we need an unsafe here instead of just on the `InvalidateSome` interface? Is it because this returns a more generic type?

Reply:
I suspect this was autogenerated: but `CommitUpdate` might not necessarily be unsafe (`InvalidateAll` is currently unannotated)

Reply:
yeah, I guess this makes sense, but surely any commit update we come across _might_ be unsafe, and so we shouldn't log it unless proven otherwise? I'd probably need to double check the code to confirm whether we can actually get that any other way (or only via this method, in which case it makes sense).

Reply:
I split up the commits into the primary one marking `LockDescriptor` as unsafe https://github.com/palantir/atlasdb/pull/6348/commits/4948d6fe2b08f0edda6f2d0e29c42c1c345a7574 , then the rest of changes are just from running [`./gradlew classes testClasses -PerrorProneApply=SafeLoggingPropagation && ./gradlew format`](https://github.com/palantir/atlasdb/pull/6348/commits/19a7a401b10eb0944f3928156de6737d5f453178)